compiler/pom.xml

```diff
@@ -32,6 +32,16 @@
       <artifactId>auto-value</artifactId>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>org.jgrapht</groupId>
```

Comment:
BTW, can you explain me more about this plugin? What it is about? Why renaming the dependencies?